MobiusCore/Source/MobiusLoop.swift

```diff
@@ -55,16 +63,19 @@ public final class MobiusLoop<Model, Event, Effect>: Disposable, CustomDebugStri
     /// - Returns: a `Disposable` that can be used to stop further notifications to the observer
     @discardableResult
     public func addObserver(_ consumer: @escaping Consumer<Model>) -> Disposable {
-        return modelPublisher.connect(to: consumer)
+        return access.guard {
```

Comment:
It is, but I don’t see a great need to bake in knowledge of whether it is or isn’t here.

Reply:
My comment came from assuming that the `guard` actually prevented races, rather than just assert that two threads don't hit this method at the same time. For the way it currently works,  I agree.